build.gradle

```diff
@@ -44,20 +48,21 @@ configFile.withReader {
 
 dependencies {
     compile "codechicken:CodeChickenLib:${config.mc_version}-${config.ccl_version}:deobf"
-	compile "mezz.jei:jei_${config.mc_version}:${config.jei_version}"
+    compile "mezz.jei:jei_${config.mc_version}:${config.jei_version}"
+    compile 'inventory-tweaks:InventoryTweaks:1.63+beta.107:api'
 }
 
-version =  "${project.config.mod_version}." + System.getenv("BUILD_NUMBER") ?: "1"
+version = "${project.config.mod_version}." + (System.getenv("BUILD_NUMBER") ?: "1")
```

Comment:
It actually didn't work if the environment variable wasn't set. The Elvis operator in Groovy is a [lower precedence than addition](http://groovy-lang.org/operators.html#_operator_precedence), so it would produce version numbers with `null` appended to them instead of `1`.